### PR TITLE
Update readme and add start script

### PR DESCRIPTION
**Pull request by antsgar**
Updated readme with instructions to run locally and created another heading for the Docker setup instructions, is that okay? Do we want both instructions available or do we need some regular local setup instructions until Docker implementation is ready?

### CODE REVIEW

**Review by karolsojko — APPROVED** [review]

**Comment by karolsojko** [comment]
From what I saw I think we will have to switch to `yarn` since `webpacker` uses `yarn install` which creates `yarn.lock` and we already use `package-lock.json` so we have to fix the setup to use either `yarn` or `npm`, not both

File: README.md [hunk 1]
```diff
@@ -52,6 +52,20 @@ You can change the date that appears on your post, or set a [canonical URL](http
 
 ## Contributing
 
+### How to run application locally
+
+To run the application locally, after cloning the repo follow these steps:
+```
+npm install
```
Comment (karolsojko) [inline README.md]: `npm ci` is the command that respects the lock file

File: package.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
     "webpack-merge": "^4.2.2"
   },
   "scripts": {
+    "start": "bin/webpack-dev-server & rails s",
```
Comment (karolsojko) [inline package.json]: this is more `start:dev` right ? `start` as in production would be `bundle exec rails server -b 0.0.0.0` if I'm not wrong?

**Review by karolsojko — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (antsgar) [reply package.json]: That's a good point, I'll add a start:production script too. From what I read bundle exec rails server and rails server are the same, apparently the rails command already has integrated support to check which version it should run, maybe I'm missing something?

File: README.md (hunk 1 above)
Comment (antsgar) [reply README.md]: Good catch! Will change this although if we move to yarn we should probably change it to yarn install --frozen-lockfile?

**Comment by antsgar** [comment]
> From what I saw I think we will have to switch to `yarn` since `webpacker` uses `yarn install` which creates `yarn.lock` and we already use `package-lock.json` so we have to fix the setup to use either `yarn` or `npm`, not both

@karolsojko seems easier to use yarn, although from what I've been reading looks like there is a way to config Webpacker to use npm. What do you think? Move to Yarn?

File: README.md (hunk 1 above)
Comment (karolsojko) [reply README.md]: Yes, `yarn install --frozen-lockfile` should be the way to go

File: package.json (hunk 2 above)
Comment (karolsojko) [reply package.json]: yeah they are the same :)

**Comment by karolsojko** [comment]
> @karolsojko seems easier to use yarn, although from what I've been reading looks like there is a way to config Webpacker to use npm. What do you think? Move to Yarn?

I vote for going with yarn :) Also please remember to update the `Dockerfile` to use `yarn` instead of `npm`

**Comment by antsgar** [comment]
@karolsojko changed npm to yarn and updated Dockerfile as well, please check it out and let me know if any changes are needed. Thank you!

**Review by karolsojko — APPROVED** [review]
Good job! 👍

**Pull request merged**